src/main/resources/templates/pom.tl.xml

```diff
@@ -54,16 +54,8 @@
   </build>
 
   <dependencies>
-    <!-- Java EE 7 dependency -->
-    <dependency>
-      <groupId>javax</groupId>
```

Comment:
Is this intentional? The swarm plug-in used to detect fractions, is that no longer the case?

Reply:
It is still the case, but dependencies on the fractions are added explicitly, so this is just redundant.

Reply:
Maybe add it if no dependencies were added?

Reply:
Hmm makes sense!

Reply:
Done.